doc/nrf/conf.py

```diff
@@ -80,7 +80,12 @@
 html_show_sourcelink = True
 html_show_sphinx = False
 
-html_theme_options = {"docset": "nrf", "docsets": utils.ALL_DOCSETS}
+html_theme_options = {
+    "docset": "nrf",
+    "docsets": utils.ALL_DOCSETS,
+    "add_gtm": True,
+    "gtm_id": "GTM-WF4CVFX",
```

Comment:
I guess this applies to all docsets, so maybe use `utils`?

Reply:
Good catch! Fixed, thanks!